Onboarding/Sources/Onboarding/ViewModels/OnboardingViewModel.swift

```diff
@@ -0,0 +1,84 @@
+import Foundation
+import SwiftUI
+
+  public class OnboardingViewModel: ObservableObject {
+    @Published var tabIndex: Int = 0
+    @Published var state = State.loading
+    @Published var primaryButtonTitle = "Continue"
+    let skipButtonAcessibilityHint = "Skip onboarding"
+    let lastButtonTitle: String = "Done"
+    let skipButtonTitle: String = "Skip"
+    @Published var onboardingSlidesCount: Int = 0
+    private let onbaordingService: OnboardingServiceInterface
+    private let dismissAction: () -> Void
+    private let onboardingType: OnboardingType
+
+    public init(onboardingService: OnboardingServiceInterface = OnboardingService(),
```

Comment:
Can you remove the default here given it doesn't lool like you actually use it?

Reply:
done